### PR TITLE
etcdctlv3: expose store size and raft status in 'etcdctl status' command

### PR DESCRIPTION
**Pull request by mqliang**
ref https://github.com/coreos/etcd/issues/4797

@xiang90

### CODE REVIEW

File: etcdctl/ctlv3/command/printer.go [hunk 1]
```diff
@@ -131,13 +131,17 @@ func (s *simplePrinter) MemberList(resp v3.MemberListResponse) {
 
 func (s *simplePrinter) MemberStatus(statusList []statusInfo) {
 	table := tablewriter.NewWriter(os.Stdout)
-	table.SetHeader([]string{"endpoint", "ID", "version"})
+	table.SetHeader([]string{"endpoint", "ID", "version", "store size", "raft index", "raft term", "raft lead"})
```
Comment (xiang90) [inline etcdctl/ctlv3/command/printer.go]: probably just db size? then leader, raft term, raft index? also we should use something like https://gowalker.org/github.com/dustin/go-humanize to format db size in bytes.

**Comment by xiang90** [comment]
Do you have any example output?

**Comment by mqliang** [comment]
example output:
```
➜  bin git:(memberStatus) ✗ ./etcdctl status
+-----------------+------------------+-----------+-----------+-----------+----------+----------------------+
|    ENDPOINT     |        ID        |  VERSION  | STORESIZE | RAFTINDEX | RAFTTERM |       RAFTLEAD       |
+-----------------+------------------+-----------+-----------+-----------+----------+----------------------+
| 127.0.0.1:2379  | 8211f1d0f64f3269 | 2.3.0+git |     24576 |     87086 |        6 | 18249187646912138824 |
| 127.0.0.1:22379 | 91bc3c398fb3c146 | 2.3.0+git |     24576 |     87086 |        6 | 18249187646912138824 |
| 127.0.0.1:32379 | fd422379fda50e48 | 2.3.0+git |     24576 |     87086 |        6 | 18249187646912138824 |
+-----------------+------------------+-----------+-----------+-----------+----------+----------------------+

```

File: etcdctl/ctlv3/command/printer.go (hunk 1 above)
Comment (mqliang) [inline etcdctl/ctlv3/command/printer.go]: Maybe output "Is leader"? and revert https://github.com/coreos/etcd/pull/4777

Comment (xiang90) [inline etcdctl/ctlv3/command/printer.go]: yea.. that is what i was thinking. just move the raft leader col and rename it to isLead

**Comment by mqliang** [comment]
@xiang90 Updated. PTAL. Sample output:

```
➜  bin git:(memberStatus) ✗ ./etcdctl status
+-----------------+------------------+-----------+---------+-----------+-----------+------------+
|    ENDPOINT     |        ID        |  VERSION  | DB SIZE | IS LEADER | RAFT TERM | RAFT INDEX |
+-----------------+------------------+-----------+---------+-----------+-----------+------------+
| 127.0.0.1:2379  | 8211f1d0f64f3269 | 2.3.0+git | 25 kB   | false     |         9 |     100080 |
| 127.0.0.1:22379 | 91bc3c398fb3c146 | 2.3.0+git | 25 kB   | false     |         9 |     100080 |
| 127.0.0.1:32379 | fd422379fda50e48 | 2.3.0+git | 25 kB   | true      |         9 |     100080 |
+-----------------+------------------+-----------+---------+-----------+-----------+------------+
```

**Comment by xiang90** [comment]
@mqliang Can you please update the commit message of the last message to `etcdctl: print db size and raft status in 'etcdctl status' command` (add etcdctl as subpkg in commit message)

**Comment by mqliang** [comment]
@xiang90 Reworked. Sorry for my missing.

**Comment by xiang90** [comment]
@mqliang gofmt fails (https://semaphoreci.com/coreos/etcd/branches/pull-request-4957/builds/3).

**Comment by mqliang** [comment]
@xiang90 
```
Checking gofmt...
Checking govet...
Checking govet -shadow...
Checking goword...
./test: line 104: goword: command not found
Checking for license header...
license header checking failed:
  ./vendor/golang.org/x/tools/present/html.go
  ./vendor/golang.org/x/tools/oracle/pos.go
  ./vendor/golang.org/x/tools/imports/mkstdlib.go
  ./vendor/golang.org/x/tools/go/callgraph/static/static.go
  ./vendor/golang.org/x/tools/go/buildutil/tags.go
  ./vendor/golang.org/x/tools/go/buildutil/fakecontext.go
  ./vendor/golang.org/x/tools/go/ast/astutil/util.go
  ./vendor/github.com/gogo/protobuf/protoc-gen-gogo/descriptor/gostring.go
  ./vendor/github.com/dustin/go-humanize/times.go
  ./vendor/github.com/dustin/go-humanize/si.go
  ./vendor/github.com/dustin/go-humanize/ordinals.go
  ./vendor/github.com/dustin/go-humanize/number.go
  ./vendor/github.com/dustin/go-humanize/humanize.go
  ./vendor/github.com/dustin/go-humanize/ftoa.go
  ./vendor/github.com/dustin/go-humanize/comma.go
  ./vendor/github.com/dustin/go-humanize/bytes.go
  ./vendor/github.com/dustin/go-humanize/bigbytes.go
  ./vendor/github.com/dustin/go-humanize/big.go
```

Some files I import have no license header. Any  suggestion?

**Comment by xiang90** [comment]
@mqliang I think there is something wrong with your godep at the first place. this pr introduces tons of dependencies which it should not. We recently changed the repo layout. Probably you need to do a rebase. Then you will need to cd into cmd dir to vendor dependency again. Sorry for the inconvenience!

**Comment by heyitsanthony** [comment]
@xiang90 @mqliang godep will mess up with `cd cmd && godep save ./...`. It did work like that in the middle of the revendor PR but unfortunately it seems like it doesn't now; it tries to vendor etcd.

My workaround right now is:
```sh
# from etcd project root
ln -s cmd/vendor vendor
godep save ./cmd/...
rm vendor
rm -rf cmd/Godeps
mv Godeps cmd/
```

**Comment by xiang90** [comment]
@mqliang I just introduced the dependency here: https://github.com/coreos/etcd/pull/5041. So just rebase with master, then everything should be OK.

**Comment by xiang90** [comment]
Thanks!

**Pull request merged**